docs/src/P3Scheme.md

```diff
@@ -1,225 +1,351 @@
+    |graupel (completely rimed, spherical) | $D_{gr} < D < D_{cr}$ | $L_{rim} > 0$  | $\frac{π}{6} ρ_g D^3$                 | $G_4 = G\left(D_{gr},D_{cr},\frac{π}{6} ρ_g,3,μ,λ\right)$           |
+    |partially rimed ice                   | $D_{cr} < D$          | $L_{rim} > 0$  | $\frac{α_{va}}{1-F_{rim}} D^{β_{va}}$ | $G_5 = G\left(D_{cr},∞,\frac{α_{va}}{1-F_{rim}},β_{va},μ,λ\right)$  |
+
+    Let LSE denote the log-sum-exp operation: $\mathrm{LSE}(x_1,\ldots,x_n) = \log\sum_{i=1}^n \exp(x_i)$. Then,
+
+    - If $L_\mathrm{rim} = 0$:
+
+    ```math
+    \log ∫_{0}^{∞} m(D) D^μ e^{-λD}\ \mathrm{d}D = \mathrm{LSE}(G_1, G_2)
+    ```
+
+    -  If $L_\mathrm{rim} > 0$:
+
+    ```math
+    \log ∫_{0}^{∞} m(D) D^μ e^{-λD}\ \mathrm{d}D = \mathrm{LSE}(G_1, G_3, G_4, G_5)
+    ```
+
+    The LSE operation is computed in a numerically stable manner using the `logsumexp` function from the `LogExpFunctions.jl` package.
+
+
+### Parameterizations for the slope parameter $μ$
+
+#### $μ$ as a power law in $λ$
+
+The de-facto standard parameterization for $μ$ as a function of $λ$ is a power law on the form
+
+```math
+μ = a λ^b - c
 ```
-![](P3_GammaInc_error.svg)
 
-In our solver, we approximate ``\mu`` from ``L/N`` and keep it constant throughout the solving step.
-We approximate ``\mu`` by an exponential function given by the ``L/N`` points
-  corresponding to ``\mu = 6`` and ``\mu = 0``.
-This is shown below as well as how this affects the solvers ``\lambda`` solutions.
+typically with lower and upper limiters. In [MorrisonMilbrandt2015](@cite), the following coefficients and limiting behavior are used:
+
+```math
+μ = \begin{cases}
+    0 & \text{if } μ < 0 \\
+    0.00191 λ^{0.8} - 2 & \text{if } 0 ≤ μ ≤ 6 \\
+    6 & \text{if } μ > 6
+\end{cases}
+```
+which looks like this:
 
 ```@example
-include("plots/P3LambdaErrorPlots.jl")
+include("plots/P3SlopeParameterizations.jl")
 ```
-![](MuApprox.svg)
+![](P3SlopeParameterizations_power_law.svg)
 
-An initial guess for the non-linear solver is found by approximating the gamma functions
-  as a simple linear function from ``x = \log{(L/N)}`` to ``y = \log{(\lambda)}``.
-The equation is given by ``(x - x_1) = A \; (y - y_1)`` where
-  ``A = \frac{x_1 - x_2}{y_1 - y_2}``.
-``y_1`` and ``y_2`` define ``log(\lambda)`` values for three ``L/N`` ranges
+With this choice, it appears that some values of $\log(L/N)$ gives rise to multiple solutions for $λ$, as seen in the plot below.
 
-|             L/N                |         ``y_1``        |      ``y_2``         |
-|:-------------------------------|:-----------------------|:---------------------|
-|        ``L/N >= 10^-8``        |         ``1``          |     ``6 * 10^3``     |
-|   ``2 * 10^9 <= L/N < 10^-8``  |     ``6 * 10^3``       |     ``3 * 10^4``     |
-|        ``L/N < 2 * 10^9``      |     ``4 * 10^4``       |       ``10^6``       |
+!!! todo "TODO: Add plot of multiple solutions"
```

Comment:
That would be nice to add

Reply:
Added to backlog. Will do in next PR, ref #529